ryvencore-qt/ryvencore_qt/src/flows/FlowView.py

```diff
@@ -1308,12 +1308,17 @@ def selected_drawings(self) -> [DrawingObject]:
     def select_items(self, items: list):
         """
         Selects a list of items in the scene.
+        It always clears the current selection
         Does not cause any command to be pushed to the undo stack.
         Emits signals indicating selection changes.
         """
 
         self._unwatch_scene_selection()
         self._current_selected = items
+        for i in self.scene().items():
+            if i.ItemIsSelectable:
```

Comment:
I don't think this is correct. You are accessing `QGraphicsItem.ItemIsSelectable` which is a const flag, and then you cast that to a `bool`, this result will always be the same (`True` because the flag value is not `0`). I think you want to do something like

```python
if i.flags() & QGraphicsItem.ItemIsSelectable != 0:
```

Reply:
hahah careful with source history, you might find dark corners in this repo